modules/qna/src/backend/transfer.ts

```diff
@@ -16,7 +59,14 @@ export const importQuestions = async (questions: QnaItem[], storage, statusCallb
   })
 }
 
-export const prepareExport = async (storage: Storage) => {
+export const prepareExport = async (storage: Storage, bp: typeof sdk) => {
   const qnas = await storage.fetchQNAs()
-  return JSON.stringify(qnas, undefined, 2)
+  const contentElementIds = await storage.getAllContentElementIds()
+
+  const cms = await Promise.mapSeries(contentElementIds, async id => {
```

Comment:
this is fine, I'm not just sure about the naming .. would cmsItems ? or contentElements? be more appropriate ?

cms could feel like a cms export